app/Helpers/MapHelper.php

```diff
@@ -46,8 +46,12 @@ public static function getAddressAsString(Address $address): ?string
     {
         $sentence = '';
 
-        if (! is_null($address->street)) {
-            $sentence = $address->street;
+        if (! is_null($address->line_1)) {
+            $sentence = $address->line_1;
+        }
+
+        if (! is_null($address->line_2)) {
+            $sentence = $address->line_2;
```

Comment:
You are missing something here.
It should be `$sentence .= ' ' . $address->line_2;`, so the line_2 is concatenated with the pre-existing sentence that could have been filled with line_1 already.